Tests/OpenCombineTests/PublisherTests/DecodeTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DecodeTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 6/21/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class DecodeTests: XCTestCase {
+    static let allTests = [
+        ("testDecodeWorks", testDecodeWorks)
+    ]
+    
+    private let jsonEncoder = JSONEncoder()
+    private let jsonDecoder = JSONDecoder()
+    
+    var cancel: Cancellable?
```

Comment:
Not sure, why should we make this a stored property? IMO to guarantee the independence of the tests, it's better to move this var into the test method.